integration_tests/Cargo.toml

```diff
@@ -9,7 +9,7 @@ rust-version.workspace = true
 version.workspace = true
 
 [features]
-kube-openssl = ["kube/openssl-tls"]
+kube-openssl = []
```

Comment:
nit: add comment explaining why this feature still exists/at which point it may be removed.